src/mol-repr/structure/visual/util/bond.ts

```diff
@@ -20,6 +20,7 @@ export const BondParams = {
     includeTypes: PD.MultiSelect(ObjectKeys(BondType.Names), PD.objectToOptions(BondType.Names)),
     excludeTypes: PD.MultiSelect([] as BondType.Names[], PD.objectToOptions(BondType.Names)),
     ignoreHydrogens: PD.Boolean(false),
+    aromaticBonds: PD.Boolean(false, { description: 'Display aromatic bonds with dashes' }),
```

Comment:
Why false by default?

Reply:
1) Don't want to change the current behavior. 2) Not sure what people want to see more, the kekulized form or the aromatics.